doc/API.md

```diff
@@ -1087,7 +1085,6 @@ Response:
 	UploadProgress float32
 	Nickname       string
 	Filesize       uint64
-	TimeRemaining  types.BlockHeight (uint64)
```

Comment:
`TimeRemaining` has been changed to `Expiration`, which is an absolute block height rather than a relative number.